ansible/roles/kubeadm/tasks/debian.yaml

```diff
@@ -2,6 +2,27 @@
 - name: remove version hold for kubeadm packages
   command: apt-mark unhold kubeadm
 
+- name: remove version hold for cri-tools package
+  command: apt-mark unhold cri-tools
+
+# cri-tools version should be, approximately, the Kubernetes version.
+# The community-maintained kubeadm package installs the latest version of
+# cri-tools, which may be incompatible with the Kubernetes version.
+# Therefore, we install a version that we know to be compatible.
+- name: install cri-tools remote deb package
+  shell: |
+    apt-get install --force-yes --yes \
+      cri-tools={{ critools_deb }}
+  args:
+    warn: false
+  register: result
+  until: result is success
+  retries: 3
+  delay: 3
+
+- name: add version hold for cri-tools package
```

Comment:
Great idea!

Reply:
Do you have any thoughts about how the two kubeadm and cri-tools holds interact? I'm a little worried the order matters, but [man page](https://manpages.ubuntu.com/manpages/bionic/man8/apt-mark.8.html) isn't enlightening.

Reply:
Good point, thanks! I added a comment explaining the hold.